forge-gui/res/cardsfolder/upcoming/vito_fanatic_of_aclazotz.txt

```diff
@@ -0,0 +1,12 @@
+Name:Vito, Fanatic of Aclazotz
+ManaCost:2 W B
+Types:Legendary Creature Vampire Demon
+PT:4/4
+K:Flying
+T:Mode$ Sacrificed | ValidCard$ Permanent.Other | ValidPlayer$ You | Execute$ TrigGainLife | TriggerZones$ Battlefield | TriggerDescription$ Whenever you sacrifice another permanent, you gain 2 life if this is the first time this ability has resolved this turn. If it's the second time, each opponent loses 2 life. If it's the third time, create a 4/3 white and black Vampire Demon creature token with flying.
```

Comment:
what StackDescriptions you mean? it's a trigger...

Reply:
You're right, it just that now it's not very clear which time it is

Reply:
Maybe a flag that adds times triggered to the Important Stack Objects (and therefore appends to end of Trigger Desc?)

Either way this is probably something to add to an issue for follow-up, nothing to hold up this PR over